### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by f-Xiaoxi**
-added fonts and colours to scss
-edited nav bar and made it sticky
-added header image with scroll spy(the link is not going to the exact position I'm hoping for though)
-edited the forms

will need to add diff nav bar to the header image

### CODE REVIEW

**Comment by barbayjuliette** [comment]
- When clicking explore should go to the full page of index
-  Once logged in, should not go again to the header image but directly to the index
- Buttons: I think it would be nice if the color was a bit more transparent when you hover, and the text stays in a shade of yellow when hovering.
- Otherwise looks good

File: app/assets/stylesheets/components/_form_legend_clear.scss [hunk 1]
```diff
@@ -8,3 +8,212 @@
...
+
+.meal-label-style:focus{
+  background-color: #FFDED6 !important;
+  border: 0px solid black  !important;
+  height:60px;
+  padding-left:20px;
+  font-family: 'Lekton', sans-serif;
+}
+
+.meal-label-style:hover{
```
Comment (barbayjuliette) [inline app/assets/stylesheets/components/_form_legend_clear.scss:172]: change meal

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -19,6 +20,7 @@
 // Variables
 $green: #D0E3CC;
```
Comment (barbayjuliette) [inline app/assets/stylesheets/application.scss:21]: Can delete the green here that I had added

**Review by barbayjuliette — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/application.scss (hunk 2 above)
Comment (f-Xiaoxi) [reply app/assets/stylesheets/application.scss]: Deleted

File: app/assets/stylesheets/components/_form_legend_clear.scss (hunk 1 above)
Comment (f-Xiaoxi) [reply app/assets/stylesheets/components/_form_legend_clear.scss]: removed

**Comment by f-Xiaoxi** [comment]
When clicking explore should go to the full page of index
-Is not going to correct position. Will figure out later
Once logged in, should not go again to the header image but directly to the index
-fixed
Buttons: I think it would be nice if the color was a bit more transparent when you hover, and the text stays in a shade of yellow when hovering.
-fixed

**Comment by barbayjuliette** [comment]
Can you also set the fonts / sizes to all the corresponding texts ? 
I mean for example in application.scss:p { font-family: $font-variable; ...}

**Comment by f-Xiaoxi** [comment]
edited

**Comment by barbayjuliette** [comment]
Oh okay, I see, I think there was a misunderstanding. When I said buttons, I meant all the buttons, the ones on the forms (log in, log out sign up). The one in the header image was perfectly fine as it was before

**Comment by f-Xiaoxi** [comment]
ok, made the green login buttons slightly transparent when hover.
I think the header button looks better green when hovered so didn't make any changes to it.

**Review by barbayjuliette — APPROVED** [review]
Looks good, you can merge it.

**Comment by f-Xiaoxi** [comment]
solved conflicts

**Pull request merged**